sdcm/cluster_k8s/__init__.py

```diff
@@ -704,6 +704,13 @@ def deploy_scylla_operator(self, pool_name: str = None) -> None:
                 readiness_timeout=5,
                 namespace=SCYLLA_OPERATOR_NAMESPACE
             )
+
+            if version.LegacyVersion(self._scylla_operator_chart_version.split("-")[0]) >= version.LegacyVersion("v1.8.0"):
```

Comment:
It must be configurable. Shlomi said we should not test it as part of the 1.8.0
So, need to have this version check and the configuration check that we really want to enable it.

Then, no need to wait for the pods readiness to change it's configs and restart it. So, need to place it before current line 701 -> `KubernetesOps.wait_for_pods_readiness(...)`

Reply:
move it, and added `k8s_serverless` option to SCT